src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -628,6 +628,48 @@ public bool Remove(TKey key)
             return false;
         }
 
+        public bool Remove(TKey key, out TValue value)
```

Comment:
Oops ok. @danmosemsft @jkotas I have added a comment above both overloads. Kindly review the same.